capstone/fitness/src/main/java/com/google/sps/servlets/CalendarServlet.java

```diff
@@ -29,71 +33,89 @@
 import java.time.LocalDateTime;   
 import java.util.*;
 
-
-
 // CalendarServlet initializes the OAuth process and does calendar functions. 
 @WebServlet("/calendar-servlet")
 public class CalendarServlet extends AbstractAppEngineAuthorizationCodeServlet {
-  private static String APPLICATION_NAME = "Marathon App";
-  private static String colorId = "4";
+  private static String APPLICATION_NAME = "GetIn' Progress";
+  private static String runningColorId = "4";
+  private static String liftingColorId = "8";
   private static long exerciseDuration = 30;  
   private static DateTimeFormatter dtf = DateTimeFormatter.ofPattern("MM/dd/YYYY");  
   Gson gson = new Gson();
   Calendar calendar; 
+  int scheduledLength = 0;
+  int timesPerWeek;
```

Comment:
Should probably be stateless meaning only defined locally so it doesn't get confused as to which user is calling it. If it can't be defined locally then you can use HttpSession(s).